### ISSUE TITLE
NS2 System.Data.SqlClient reference causes warnings

### ISSUE THREAD

**Issue by rockfordlhotka** [MarimerLLC/csla#818]
As per https://github.com/MarimerLLC/cslaforum/issues/502

```
The assembly 'System.Data.SqlClient.dll' was loaded from a different path than the provided path
provided path: /Users/user135287/Library/Caches/Xamarin/mtbs/builds/UI.iOS/4a61fb5d59d8c2875723f6d1e7f44ce3/bin/iPhoneSimulator/Debug/System.Data.SqlClient.dll
actual path: /Library/Frameworks/Xamarin.iOS.framework/Versions/11.6.1.4/lib/mono/Xamarin.iOS/Facades/System.Data.SqlClient.dll
```

**Comment by rockfordlhotka** [comment]
This problem sadly does still exist even after updating the version of `System.Data.SqlClient` referenced by CSLA. It appears that Xamarin forms (for example) pulls the dependency from a different path. 

Solving this will take further research. Fortunately it is just a warning, not an error, and has no real-world impact.

**Comment by rockfordlhotka** [comment]
This issue isn't the same, but might perhaps be related? It sounds like there are per-platform `System.Data.SqlClient` implementations, and a dummy one for NS2?

https://github.com/dotnet/corefx/issues/24229

**Comment by rockfordlhotka** [comment]
Fixed in #981
